test/unit/model/test_model_creator.py

```diff
@@ -0,0 +1,37 @@
+# SPDX-FileCopyrightText: 2017-2021 Alliander N.V. <korte.termijn.prognoses@alliander.com> # noqa E501>
+#
+# SPDX-License-Identifier: MPL-2.0
+
+from unittest import TestCase
+import pytest
+from xgboost import XGBRegressor
+
+from openstf.model.xgb_quantile import XgbQuantile
+
+from openstf.model.model_creator import ModelCreator
+
+
+class TestModelCreator(TestCase):
+    def setUp(self) -> None:
+        self.pj = {"model": "xgb", "quantiles": [0.5, 0.2, 0.5]}
+
+    def test_happy_flow(self):
+        # Test happy flow
+        model = ModelCreator.create_model(self.pj)
+
+        self.assertIsInstance(model, XGBRegressor)
+
+    def test_quantile_model(self):
+        # Test if quantile model is properly returned
+        self.pj["model"] = "xgb_quantile"
+        # Create relevant model
+        model = ModelCreator.create_model(self.pj)
+
+        self.assertIsInstance(model, XgbQuantile)
+        self.assertEqual(model.quantiles, tuple(self.pj["quantiles"]))
+
+    def test_unknown_model(self):
```

Comment:
to me this is confusion. The `model` prooperty of a prediction job is actually the model type. I think we should use that name to prevent confusion.

Reply:
yes, but this also requires renaming the field in the mysql database

Reply:
We don't have to do everything at once. We can do the conversion first in the database interface and later in the database if we want.